conda_forge_tick/utils.py

```diff
@@ -46,8 +46,10 @@
     os=os,
     environ=defaultdict(str),
     compiler=lambda x: x + "_compiler_stub",
-    pin_subpackage=lambda *args, **kwargs: "subpackage_stub",
-    pin_compatible=lambda *args, **kwargs: "compatible_pin_stub",
+    # The `max_pin, ` stub is so we know when people used the functions
+    # to create the pins
+    pin_subpackage=lambda *args, **kwargs: {"package_name": args[0], **kwargs},
+    pin_compatible=lambda *args, **kwargs: {"package_name": args[0], **kwargs},
```

Comment:
Does this actually work, or do you need to dump to json and then reparse when reading? I am surprised jinja2 just let's this all go by without a hitch.

Reply:
Or maybe dump to json and then let yaml parse it? Or maybe that is what is already happening?

Reply:
This does work, `{k:v}` is valid yaml syntax for a dictionary.